exercises/C-variables/exercise.js

```diff
@@ -1,3 +1,3 @@
 // Start by creating a variable `greeting`
-
-console.log(greeting);
+let greeting = "Isn't good day at all!"
+console.log(greeting);
```

Comment:
Hi Alex,

With this exercise I believe the aim was to print it 3 times. From this I understand you should have had 3 console.log greetings?

I hope the day went better after :)